clang/test/CAS/fdepscan-result-cache-output-path.c

```diff
@@ -0,0 +1,15 @@
+// TODO: Test should be updated to not depend on a working cache at default location, which is out of test/build directory.
+//
+/// Test absolute path output.
+// RUN: rm -rf %t && mkdir %t
+// RUN: %clang -target x86_64-apple-macos11 -fdepscan=daemon -c -x c %s -o %t/out.o
```

Comment:
Can we use tests `-fdepscan=inline` here to avoid unnecessary extra processes when running tests?